bootstrap.sh

```diff
@@ -98,7 +98,6 @@ git clone -b master --single-branch https://github.com/YubicoLabs/yubikit-ios/ $
 
 mkdir -p $SRCDIR/$YUBIKIT_OUT/include
 pushd $YUBIKIT_DIR/YubiKit
-git checkout 2.0.1
```

Comment:
I'll be pinning the change to yubikit release candidate, removed this temporarily since there are some fixes that need to be merged for the error handling to work correctly.

Reply:
@iccub - Pinned to v3.0.0-Preview here: https://github.com/brave/brave-ios/pull/1939/files#diff-3af436e6760a0c5f1e3d16a3384853a8R97